### ISSUE TITLE
No change in validation loss and validation accuracy and are zero val_loss: 0.0000e+00 - val_accuracy: 0.0000e+00

### ISSUE THREAD

**Issue by AnkitaBeniwal** [ozora-ogino/LCNN#5]
In the "model.fit " function I tried setting shuffle=True. 
But still loss: 0.2522 - accuracy: 0.8989 - 
val_loss: 0.0000e+00 - val_accuracy: 0.0000e+00 <-- (PS: Classification problem)
model.compile(
        optimizer=Adam(learning_rate=lr),
        loss="sparse_categorical_crossentropy",
        metrics=["accuracy"]
     )
     model.fit(
        x_train,
        y_train,
        epochs=epochs,
        batch_size=batch_size,
        validation_data=[x_val, y_val],
        shuffle=True,
        callbacks=[es, cp_cb])
![Screenshot 2021-10-19 at 12 50 20 PM](https://user-images.githubusercontent.com/34271099/137864068-933113e7-3d76-4669-a234-4db9e3b1a37f.png)
